src/files-card-body.jsx

```diff
@@ -342,94 +351,95 @@ export const FilesCardBody = ({
-                                  key={file.name}
-                                  isSelected={!!selected.find(s => s.name === file.name)}
-                                  isGrid={isGrid}
-                                />)}
-                        </Gallery>
-                    </CardBody>}
-                {!isGrid &&
-                    <ListingTable
-                      id="folder-view"
-                      className="pf-m-no-border-rows"
-                      variant="compact"
-                      columns={[_("Name")]}
-                      rows={sortedFiles.map(file => ({
-                          columns: [
-                              {
-                                  title: (
-                                      <Item
-                                        file={file}
-                                        key={file.name}
-                                        isSelected={!!selected.find(s => s.name === file.name)}
-                                        isGrid={isGrid}
-                                      />)
-                              }
-                          ]
-                      }))}
-                    />}
-            </div>
+            {sortedFiles.length !== 0 &&
+                <Table
+                  id="folder-view"
+                  className={`pf-m-no-border-rows fileview ${isGrid ? 'view-grid' : 'view-details'}`}
+                  variant="compact"
+                >
+                    <Thead>
+                        <Tr>
+                            <Th sort={sortColumn(0)} className="col-name">{_("Name")}</Th>
+                            <Th sort={sortColumn(1)} className="col-size">{_("Size")}</Th>
+                            <Th
+                              sort={sortColumn(2)} className="col-date"
+                              modifier="nowrap"
+                            >{_("Modified")}
+                            </Th>
+                        </Tr>
+                    </Thead>
+                    <Tbody>
+                        {sortedFiles.map((file, rowIndex) =>
+                            <Row
+                              key={rowIndex}
+                              file={file}
+                              isSelected={selected.some(s => s.name === file.name)}
+                            />)}
+                    </Tbody>
+                </Table>}
         </div>
     );
 };
 
-// Memoize the Item component as rendering thousands of them on each render of parent component is costly.
-const Item = React.memo(function Item({ file, isSelected, isGrid }) {
-    function getFileType(file) {
-        if (file.type === "dir") {
-            return "directory-item";
-        } else if (file.type === "lnk" && file?.to === "dir") {
-            return "directory-item";
-        } else {
-            return "file-item";
-        }
+const getFileType = (file) => {
+    if (file.type === "dir" || file.to === "dir") {
```

Comment:
Did a simple `file.to === 'dir'` not work?

Reply:
So you mean only `file.to`?

Reply:
Yes, like the `file.type === "dir"` part here is redundant.  If `file.type === "dir"` then `file.to` will also `=== "dir"`.